webapp/ansible/roles/service-vm/tasks/github.yml

```diff
@@ -0,0 +1,3 @@
+---
+  - name: Download Django code from Github.
+    git: repo=https://github.com/gouzouni625/okeanos-LoD.git dest=/root/okeanos-LoD
```

Comment:
That is exactly the reason. At the final state, this should clone grnet/~okeanos-LoD repository.